src/data/complex/is_R_or_C.lean

```diff
@@ -785,4 +786,8 @@ linear_isometry.norm_to_continuous_linear_map of_real_li
 
 end linear_maps
 
+instance {K : Type*} [is_R_or_C K] : cstar_ring K :=
```

Comment:
```suggestion
instance : cstar_ring K :=
```
Is the variable `K` already available in the file?

Reply:
no, I checked, it's not available globally in the file.